webpack/scenes/ContentViews/__tests__/basicContentViews.fixtures.js

```diff
@@ -0,0 +1,31 @@
+const buildContentView = id => ({
+  id,
+  composite: 'false',
+  name: `contentView${id}`,
+  environments: [],
+  repositories: [],
+  versions: [],
+  last_published: 'Not Yet Published',
+});
+
+const createBasicCVs = (amount) => {
+  const response = {
+    total: 100,
+    subtotal: 100,
+    page: 1,
+    per_page: 20,
+    error: null,
+    search: null,
+    sort: {
+      by: 'name',
+      order: 'asc',
+    },
+    results: [],
+  };
+
+  Array(amount).fill().map((_, i) => response.results.push(buildContentView(i)));
```

Comment:
Should be a `forEach` since the result is thrown away.
Actually this can be simplified a bit with 
```js
[...Array(amount).keys()]
```